cuesubmit/cuesubmit/Constants.py

```diff
@@ -38,7 +38,16 @@
 NUKE_RENDER_CMD = config.get('NUKE_RENDER_CMD', 'nuke')
 BLENDER_RENDER_CMD = config.get('BLENDER_RENDER_CMD', 'blender')
 FRAME_TOKEN = config.get('FRAME_TOKEN', '#IFRAME#')
-
+COMMAND_TOKENS = {'#ZFRAME#': 'Current frame with a padding of 4',
```

Comment:
It would be nice to add a comment here linking to the the spot in the Cuebot code where the tokens are replaced.

I believe that's here: https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/cuebot/src/main/java/com/imageworks/spcue/dispatcher/DispatchSupportService.java

And vice versa -- link to this Constants file from a comment in the Cuebot code.

This should help future developers when adding new tokens, so they are both properly supported in Cuebot and discoverable for normal users via this dialog.

Reply:
Good call, it will definitely help :)

Reply:
Comments added in both files.